keanu-python/tests/test_vertex_operations.py

```diff
@@ -135,126 +138,168 @@ def test_can_do_less_than_or_equal_to_with_vertex_on_rhs(lhs: Union[Vertex, nump
 
 ### Arithmetic
 
+# Included the identity lambda to test the interoperability between vertexes and np/primitive types
+types_to_test = [generated.ConstantDouble, generated.ConstantInteger, lambda x: x]
```

Comment:
nit: `vertex_construcors_to_test`

Reply:
Hmm, but the lambda wouldn't construct a vertex?

Reply:
Ok - no worries